stack-easybib/recipes/deploy-easybib.rb

```diff
@@ -1,7 +1,10 @@
 node['deploy'].each do |application, deploy|
   case application
   when 'easybib'
-    next unless allow_deploy(application, 'easybib', 'nginxphpapp')
+    unless allow_deploy(application, 'easybib', 'nginxphpapp') || allow_deploy(application, 'easybib', 'housekeeping')
```

Comment:
@baccenfutter Please use an array for the last param, not two `allow_deploy` calls.